### ISSUE TITLE
Display Helper: elements-of-type(html5) missing elements

### ISSUE THREAD

**Issue by adion** [Compass/compass#399]
I believe the `html5` display value is missing a few tags: http://web-design.blogs.webucator.com/2010/10/14/list-of-new-html5-tags/

Also, `dialog` isn't a valid HTML element.

**Comment by chriseppstein** [comment]
The list from that site is:

```
article, aside, audio, canvas, command, datalist, details, embed, figcaption, figure,
footer, header, hgroup, keygen, mark, meter, nav, output, progress, rp, rt, ruby,
section, source, summary, time, video, wbr
```

I think some of these are inline elements so we need to sort out which is which and then change blueprint to only reset html5 block elements.

It would be great if someone could make this patch.

**Comment by dira** [comment]
Started working on the patch.

**Comment by chriseppstein** [comment]
Fixed by 07819ccfbdcf3055d53877aa93187ce5f50df8ff
